tests/conftest.py

```diff
@@ -1735,6 +1815,14 @@ def edge_conf(default_conf):
     return conf
 
 
+@pytest.fixture(scope="function")
+def edge_conf_usdt(edge_conf):
+    edge_conf.update({
+        "stake_currency": "USDT",
+    })
+    return edge_conf
```

Comment:
IF you decide to touch edge-tests - then i'd swap then all at once. 

While it seems to be used often at first glance, i don't think it will be too difficult to swap them all in one go.
if the outcome of this "convert tests to usdt" is that we have most of the setup-code duplicated, then it'll not bring many benefits ...